docs/decisions/0003-instructor-dashboard-rest-api-architecture.rst

```diff
@@ -0,0 +1,266 @@
+| POST   | /api/instructor/v1/courses/{course_key}/beta_testers               | student.auth                   | Partial   | /courses/{course_id}/instructor/api/bulk_beta_modify_access       | Bulk add users to beta tester role                                                           |
++--------+--------------------------------------------------------------------+--------------------------------+-----------+-------------------------------------------------------------------+----------------------------------------------------------------------------------------------+
+| DELETE | /api/instructor/v1/courses/{course_key}/beta_testers/{username}    | student.auth                   | Missing   | N/A                                                               | Remove a user from beta tester role                                                          |
++--------+--------------------------------------------------------------------+--------------------------------+-----------+-------------------------------------------------------------------+----------------------------------------------------------------------------------------------+
+| GET    | /api/instructor/v1/courses/{course_key}/forum/members              | discussion, student.auth       | Partial   | /courses/{course_id}/instructor/api/list_forum_members            | List users with forum moderation roles                                                       |
++--------+--------------------------------------------------------------------+--------------------------------+-----------+-------------------------------------------------------------------+----------------------------------------------------------------------------------------------+
+| POST   | /api/instructor/v1/courses/{course_key}/forum/members              | discussion, student.auth       | Partial   | /courses/{course_id}/instructor/api/update_forum_role_membership  | Grant forum moderation role to users                                                         |
++--------+--------------------------------------------------------------------+--------------------------------+-----------+-------------------------------------------------------------------+----------------------------------------------------------------------------------------------+
+| DELETE | /api/instructor/v1/courses/{course_key}/forum/members              | discussion, student.auth       | Missing   | N/A                                                               | Revoke forum moderation role from users                                                      |
++--------+--------------------------------------------------------------------+--------------------------------+-----------+-------------------------------------------------------------------+----------------------------------------------------------------------------------------------+
+
+Student Administration API
```

Comment:
Was there some decision to call these "students" instead of "learners"? The term "learner" has always been preferred in the past, and that seems like a major change!

Reply:
Hmm, you're right that there's a lot of `student` in the codebase, though it's not user-facing. More than I remembered in fact :p

I do see this as an opportunity to align the backend names with the frontend names. It's like our ongoing effort in Authoring MFE to call things "Units", "subsections" and "sections" (what users see) instead of "verticals", "sequentials", and "Chapter". Whenever we add large new API surfaces, it's an opportunity to handle naming more consistently. So I think it's best to use "learner" here since that's what it will say in the UI.